spec/java/util/LinkedList.DescendingIterator.lsl

```diff
@@ -0,0 +1,156 @@
+libsl "1.1.0";
+
+library std
+    version "11"
+    language "Java"
+    url "https://github.com/openjdk/jdk11/blob/master/src/java.base/share/classes/java/util/LinkedList.java";
+
+// imports
+
+import java/lang/Object;
+
+import java/util/LinkedList;
+
+
+// automata
+
+automaton LinkedList_DescendingIteratorAutomaton
+(
+    var parent: LinkedList,
+    var cursor: int,
+    var expectedModCount: int
+)
+: LinkedList_DescendingIterator
+{
+    // states and shifts
+
+    initstate Initialized;
+
+    shift Initialized -> self by [
+       forEachRemaining,
+       hasNext,
+       next,
+       remove,
+    ];
+
+
+    // local variables
+
+    var lastRet: int = -1;
+
+
+    // utilities
+
+    @AutoInline @Phantom proc _throwCME (): void
+    {
+        action THROW_NEW("java.util.ConcurrentModificationException", []);
+    }
+
+
+    proc _checkForComodification (): void
+    {
+        val modCount: int = LinkedListAutomaton(this.parent).modCount;
+        if (modCount != this.expectedModCount)
```

Comment:
LinkedListAutomaton(this.parent).modCount != this.expectedModCount ?

Reply:
Fixed.